docs/UserGuide.md

```diff
@@ -209,6 +209,37 @@ Examples: `deleteTagType GPA`
 
 - Deletes the `GPA` tag type.
 
+### Adding a skill : skill
+
+Adds a skill  to the person
+
+Format: `skill INDEX s/SKILL`
+* The INDEX refers to the index number shown in the displayed person list.
+* The index must be a positive integer 1, 2, 3, …​
+
+<div markdown="span" class="alert alert-primary">:bulb: **Note:**
+A person can have any number of skills (including 0)
+</div>
+
+Examples:
+* `skill 1 s/Java` adds a skill `Java` to the 1st person in the address book.
+* `skill 2 s/Python` adds a skill `Python` to the 2nd person in the address book.
+
```

Comment:
In our last meeting, we had decided that skills won't be a separate command, and would be part of the add tag command itself, so perhaps we can get rid of this?